luadox/render/base.py

```diff
@@ -32,15 +32,15 @@ def __init__(self, parser: Parser):
         self.ctx = parser.ctx
 
     def copy_file_from_config(self, section: str, option: str, outdir: str) -> None:
-        fname = self.config.get(section, option, fallback=None)
-        if not fname:
+        fnames = self.config.get(section, option, fallback='').split()
```

Comment:
This doesn't support filenames with spaces.  We need to use here the same logic that's in [get_files() in main.py](https://github.com/jtackaberry/luadox/blob/a5378e4f6d2db769893a9b047e085e8aeb96d7a7/luadox/main.py#L156-L158) -- don't need to support globs, but do need splitlines() instead of split(), and then followed up by use of shlex.split() to support quoted/escaped spaces.